docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+git+https://github.com/machow/siuba.git@stable
```

Comment:
does this work in a reqs.txt?

Reply:
> does this work in a reqs.txt?

it did work for me, I was able to build and install siuba from `docs/requirements.txt` in my venv... do you think it's reliable enough to keep though @atvaccaro ?